lib/onelogin/ruby-saml/idp_metadata_parser.rb

```diff
@@ -62,7 +63,14 @@ def get_idp_metadata(url, validate_cert)
           get = Net::HTTP::Get.new(uri.request_uri)
           response = http.request(get)
           meta_text = response.body
+        elsif uri.scheme == nil
```

Comment:
actually, it should probably just be an `else` so we catch all cases. i'll make that change